openmc/settings.py

```diff
@@ -101,6 +101,13 @@ class Settings(object):
         Coordinates of the lower-left point of the Shannon entropy mesh
     entropy_upper_right : tuple or list
         Coordinates of the upper-right point of the Shannon entropy mesh
+    tabular_legendre : dict
```

Comment:
Maybe split this into two attributes - `legendre_to_tabular` and `legendre_to_tabular_points` - to be consistent with the variables in `global.F90`.